components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/data/containers_tree.html

```diff
@@ -117,15 +117,15 @@
         {% if share.shSize %}
             {% for s in share.shares %}
             {% if s.isEmpty %}
-            <li id='discussion-{{ s.id }}' rel="discussion{% if not s.canEdit %}-locked{% endif %}"><a href="#">{{ s.id }}</a></li>
+            <li id='discussion-{{ s.id }}' rel="discussion" class="{% if s.canEdit %}canEdit{% endif %}"><a href="#">{{ s.id }} {{ s.uuid }}</a></li>
             {% else %}
                 {% if s.isOwned %}
-                <li id='share-{{ s.id }}' rel="share" class="jstree-closed"><a href="#">{{ s.id }}</a></li>
+                <li id='share-{{ s.id }}' rel="share" class="jstree-closed"><a href="#">{{ s.id }} {{ s.uuid }}</a></li>
                 {% else %}
                     {% if s.active %}
-                    <li id='share-{{ s.id }}' rel="share-locked" class="jstree-closed"><a href="#">{{ s.id }}</a></li>
+                    <li id='share-{{ s.id }}' rel="share" class="jstree-closed {% if s.canEdit %}canEdit{% endif %}"><a href="#">{{ s.id }} {{ s.uuid }}</a></li>
                     {% else %}
-                    <li id='share-{{ s.id }}' rel="share-disabled"><a href="#">{{ s.id }} DISABLED</a></li>
+                    <li id='share-{{ s.id }}' rel="share" class="disabled"><a href="#">{{ s.id }} {{ s.uuid }} DISABLED</a></li>
```

Comment:
Was there a reason to add ``` {{ s.uuid }} ``` to all shares in the Tree?

![screen shot 2015-06-12 at 11 05 56](https://cloud.githubusercontent.com/assets/900055/8128001/152d4100-10f3-11e5-96f4-8183d61e37da.png)